README.md

```diff
@@ -68,7 +68,7 @@ on this under License below.)
 Using your system's package manager:
 
 * **ArchLinux** `pacman -S qt5-base`
-* **Debian** `apt-get install qtbase5-dev`
+* **Debian** `brew install qt5`
```

Comment:
Brew on Debian?  You can also add MacOS in that list :)

If you want, you can work on/add a chapter on using your shard.  I think that would make it much easier to find to the curious tinkerer.  Would be great if you could update the README on your end as well (Some TODOs etc)

Reply:
No this was be reading Debian as **Darwin**... ill fix it up.

Reply:
but also interesting enough, there is brew for linux: https://docs.brew.sh/Homebrew-on-Linux